staging/src/k8s.io/apiserver/pkg/storage/cacher/store_btree.go

```diff
@@ -435,6 +435,14 @@ func newStoreSnapshotter() *storeSnapshotter {
 	return s
 }
 
+type Snapshotter interface {
```

Comment:
nit: introduction of this interface also seem orthogonal to this change - can we also move it to a separate PR
[might be the same one I asked for in the context of test changes]

Reply:
Extracted https://github.com/kubernetes/kubernetes/pull/130925